v3.3/glfw/window.go

```diff
@@ -809,7 +809,7 @@ func (w *Window) SetCloseCallback(cbfun CloseCallback) (previous CloseCallback)
 
 // MaximizeCallback is the function signature for window maximize callback
 // functions.
-type MaximizeCallback func(w *Window, iconified bool)
+type MaximizeCallback func(w *Window, maximized bool)
```

Comment:
If you'd like, lines 174, 229, and 231 can also benefit from a similar fix.

Reply:
Thanks! I have updated the commit.